src/createAction.lua

```diff
@@ -0,0 +1,6 @@
+return function(actionType, payload)
+	return {
+		type = actionType,
+		payload = payload,
+	}
+end
```

Comment:
Planning to integrate Rodux, or just using Rodux-esque principles?

Reply:
🤷‍♂ This was in the original implementation

Reply:
LOL nevermind then